packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss

```diff
@@ -53,6 +49,11 @@ $progress-ring-size: var(--progress-ring-size, 24px);
   display: flex;
   flex-direction: column;
   border-radius: 8px;
+
+  &.compact,
+  &.fullView {
+    padding-inline-start: 10px;
+  }
```

Comment:
This sets a non-uniform paddin in fullView mode of the file list tab:
![image](https://user-images.githubusercontent.com/8081536/228572804-73198b5c-52ca-4509-b5ee-b7e0a62dbfc1.png)

So when you hover on files, left side has a padding.
![image](https://user-images.githubusercontent.com/8081536/228573270-e8b2baec-cebc-4e94-94a8-e1effd2d1eae.png)
![image](https://user-images.githubusercontent.com/8081536/228573497-ded1bf51-a7fd-463e-98cd-3c429069c040.png)

I think it should be uniform. You could remove the padding or set a left and right padding.